packages/safe-ds-lang/src/resources/builtins/safeds/lang/ideIntegration.sdsstub

```diff
@@ -22,37 +22,61 @@ annotation Category(category: DataScienceCategory)
 /**
  * A step in the data science process.
  */
-@Experimental
 enum DataScienceCategory {
+    /* Subcategories are formatted like this: CategoryQSubcategory as other symboles are currently disallowed */
+
+    /**
+     * Basic building blocks.
+     */
+    BasicElement
+
     /**
      * Load your data into the program.
      */
     DataImport
 
-    /**
-     * Prepare your data for analysis.
+     /**
+     * Save your data for later use.
      */
-    DataPreparation
+    DataExport
 
     /**
      * Explore your data.
      */
-    DataExploration
+    DataExplorationQMetric
+    DataExplorationQVisualization
+    DataExplorationQGeneral
 
     /**
-     * Save your data for later use.
+     * Prepare your data for analysis.
      */
-    DataExport
+    DataProcessingQColumn
```

Comment:
What does the "Q" stand for? Just some separator?

Reply:
Indeed. Special Characters are not allowed, so I decided to use 'Q' as a Separator.